API Project/src/main/java/com/bounswe/services/CulturalHeritageService.java

```diff
@@ -35,5 +35,10 @@ public void save(CulturalHeritage culturalHeritage) {
     while(it.hasNext()) culturalHeritages.add(it.next());
     return culturalHeritages;
   }
+
+  public void delete(CulturalHeritage culturalHeritage){
+	this.culturalHeritageRepository.delete(culturalHeritage);
```

Comment:
I did not test it, I test now but in documentation delete method has another version that takes entity as paramater.